tests/xcm-simulator/src/tests/experimental.rs

```diff
@@ -240,3 +240,141 @@ fn xcm_remote_transact_contract() {
         assert_eq!(res, Ok(false));
     });
 }
+
+#[test]
+fn test_async_xcm_contract_call_no_ce() {
+    /// All the fees and weights values required for the whole
+    /// operation.
+    #[derive(Encode)]
+    pub struct WeightsAndFees {
```

Comment:
We cannot since that will also add `ink` crates dependency on simulator which is not good, as it might cause conflict with `panic!` and allocator impl as both `ink-allocator` and `sp-io` has same implementations for global allocator and panic routines.
We can in future (when we have more contracts in fixtures) make a common crate to share types between contracts and simulator.